recipes/pdm-backend/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pdm-backend" %}
+{% set version = "2.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pdm_backend-{{ version }}.tar.gz
+  sha256: a118f1d8553d279b732a9180adc522bd8a973a258e3d4957a76a0cef0d417819
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
```

Comment:
Upstream has
```
dependencies = [
    "importlib-metadata>=3.6; python_version < \"3.10\""
]
```
is this really going to work without it?

Reply:
copy-pasted from pdm-pep517 which also had that discrepancy. I assume so (but maybe worth asking the maintainer)

Reply:
@hadim

Reply:
I don't recall. I suggest removing it?

Reply:
Little bit of googling and it looks like importlib-metadata was actually added to stdlib in 3.8. As conda-forge dropped support for 3.7 last year, removing it is fine.

Reply:
`importlib-metadata` v1.4 was added to stdlib in 3.8. So there could be a problem if it required anything introduced in 3.10 which was `importlib-metadata` v4.6